views/account/reset.pug

```diff
@@ -8,11 +8,11 @@ block content
     .form-group.row
       label.col-md-3.col-form-label.font-weight-bold.text-right(for='password') New Password
       .col-md-7
-        input.form-control(type='password', name='password', id='password', placeholder='New password', autofocus, required)
+        input.form-control(type='password', name='password', id='password', placeholder='New password', autofocus, autocomplete="new-password", required)
     .form-group.row
       label.col-md-3.col-form-label.font-weight-bold.text-right(for='confirm') Confirm Password
       .col-md-7
-        input.form-control(type='password', name='confirm', id='confirm', placeholder='Confirm password', required)
+        input.form-control(type='password', name='confirm', id='confirm', placeholder='Confirm password', autocomplete, required)
```

Comment:
I think this line needs the same new-password param.

Reply:
agreed, done